torch/autograd/function.py

```diff
@@ -334,6 +334,9 @@ def __init__(cls, name, bases, attrs):
         backward_fn._compiled_autograd_should_lift = attrs.get(  # type: ignore[attr-defined]
             "_compiled_autograd_should_lift", True
         )
+        backward_fn._bw_module = None  # type: ignore[attr-defined]
+        if getattr(cls, "_lazy_backward_info", None):
```

Comment:
do we want the `autograd.Function` code to know about these private compiler attributes? (mostly worried about someone changing the name of e.g. `_lazy_backward_info` and not realizing they need to change this code too, since it will fail silently)

Reply:
It should fail loudly due to the compiled autograd tests? Taking suggestions (I can take this as a followup)